v2/bundle/jwtbundle/bundle.go

```diff
@@ -1,68 +1,67 @@
 package jwtbundle
 
 import (
-	"bytes"
 	"crypto"
 	"encoding/json"
-	"errors"
-	"fmt"
 	"io"
 	"io/ioutil"
 	"sync"
 
 	"github.com/spiffe/go-spiffe/v2/spiffeid"
+	"github.com/zeebo/errs"
 	"gopkg.in/square/go-jose.v2"
 )
 
+var (
+	jwtbundleErr = errs.Class("jwtbundle")
+)
+
 // Bundle is a collection of trusted JWT public keys for a trust domain.
 type Bundle struct {
-	mtx            sync.RWMutex
-	trustDomain    spiffeid.TrustDomain
-	jwtSigningKeys map[string]crypto.PublicKey
+	mtx         sync.RWMutex
+	trustDomain spiffeid.TrustDomain
+	jwtKeys     map[string]crypto.PublicKey
```

Comment:
nit: group the fields protected by the mutex (with the mutex at the top of the group of fields protected by it)

```suggestion
	trustDomain spiffeid.TrustDomain
	
	mtx         sync.RWMutex
	jwtKeys     map[string]crypto.PublicKey
```